src/api/VoteMonitor.Entities/NoteAttachment.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+
+namespace VoteMonitor.Entities
+{
+    public class NoteAttachment
+    {
+        public NoteAttachment(string notePath)
+            => NotePath = notePath;
+
+        [Key, Required]
+        public int Id { get; set; }
+
+        [Required, MaxLength(1000)]
+        public string NotePath { get; set; }
+
+        [JsonIgnore]
+        public int NoteId { get; set; }
+
+        [JsonIgnore]
+        public Note Note { get; set; }
```

Comment:
should this be `virtual`?

Reply:
Lazy loading wasn't enabled by default thus adding virtual wouldn't have helped.
I've done the additional steps of enabling Lazy Loading by installing **Microsoft.EntityFrameworkCore.Proxies** nuget and enabling it from our **AddDbContext**

https://docs.microsoft.com/en-us/ef/core/querying/related-data